### PR TITLE
Updating old cards

### PR DESCRIPTION
**Pull request by mcmillanmajora**
Updated the cards for [Allocine](https://github.com/mcmillanmajora/datasets/tree/updating-old-cards/datasets/allocine), [CNN/DailyMail](https://github.com/mcmillanmajora/datasets/tree/updating-old-cards/datasets/cnn_dailymail), and [SNLI](https://github.com/mcmillanmajora/datasets/tree/updating-old-cards/datasets/snli). For the most part, the information was just rearranged or rephrased, but the social impact statements are new.

### CODE REVIEW

File: datasets/allocine/README.md [hunk 1]
```diff
@@ -22,106 +22,147 @@ task_ids:
...
 
+### Citation Information
+
+> Théophile Blard, French sentiment analysis with BERT, (2020), GitHub repository, <https://github.com/TheophileBlard/french-sentiment-analysis-with-bert>
 
 ### Contributions
 
-Thanks to [@thomwolf](https://github.com/thomwolf), [@TheophileBlard](https://github.com/TheophileBlard), [@lewtun](https://github.com/lewtun) for adding this dataset.
\ No newline at end of file
+Thanks to [@thomwolf](https://github.com/thomwolf), [@TheophileBlard](https://github.com/TheophileBlard), [@lewtun](https://github.com/lewtun) for adding this dataset.
```
Comment (lhoestq) [inline datasets/allocine/README.md:168]: You should be here as well ;)
```suggestion
Thanks to [@thomwolf](https://github.com/thomwolf), [@TheophileBlard](https://github.com/TheophileBlard), [@lewtun](https://github.com/lewtun) and [@mcmillanmajora](https://github.com/mcmillanmajora) for adding this dataset.
```

File: datasets/cnn_dailymail/README.md [hunk 2]
```diff
@@ -21,115 +21,195 @@ task_ids:
...
+  url={http://papers.nips.cc/paper/5945-teaching-machines-to-read-and-comprehend},
+  booktitle={NIPS},
+  crossref={conf/nips/2015}
+}
+
+```
+
+### Contributions
+
+Thanks to [@thomwolf](https://github.com/thomwolf), [@lewtun](https://github.com/lewtun), [@jplu](https://github.com/jplu), [@jbragg](https://github.com/jbragg), [@patrickvonplaten](https://github.com/patrickvonplaten) for adding this dataset.
```
Comment (lhoestq) [inline datasets/cnn_dailymail/README.md:214]:
```suggestion
Thanks to [@thomwolf](https://github.com/thomwolf), [@lewtun](https://github.com/lewtun), [@jplu](https://github.com/jplu), [@jbragg](https://github.com/jbragg), [@patrickvonplaten](https://github.com/patrickvonplaten) and [@mcmillanmajora](https://github.com/mcmillanmajora) for adding this dataset.

```

File: datasets/snli/README.md [hunk 3]
```diff
@@ -22,129 +22,179 @@ task_ids:
...
+	Title = {A large annotated corpus for learning natural language inference},
+	Year = {2015}
+}
+```
+
 ### Contributions
 
-Thanks to [@mariamabarham](https://github.com/mariamabarham), [@thomwolf](https://github.com/thomwolf), [@lewtun](https://github.com/lewtun), [@patrickvonplaten](https://github.com/patrickvonplaten) for adding this dataset.
\ No newline at end of file
+Thanks to [@mariamabarham](https://github.com/mariamabarham), [@thomwolf](https://github.com/thomwolf), [@lewtun](https://github.com/lewtun), [@patrickvonplaten](https://github.com/patrickvonplaten) for adding this dataset.
```
Comment (lhoestq) [inline datasets/snli/README.md:199]:
```suggestion
Thanks to [@mariamabarham](https://github.com/mariamabarham), [@thomwolf](https://github.com/thomwolf), [@lewtun](https://github.com/lewtun), [@patrickvonplaten](https://github.com/patrickvonplaten) and [@mcmillanmajora](https://github.com/mcmillanmajora) for adding this dataset.
```

**Review by lhoestq — APPROVED** [review]
Thank you so much !
Definitely the best dataset cards out here ;)

File: datasets/allocine/README.md [hunk 4]
```diff
@@ -22,106 +22,147 @@ task_ids:
...
+
+- **Homepage:** 
+- **Repository:** [Allociné dataset repository](https://github.com/TheophileBlard/french-sentiment-analysis-with-bert/tree/master/allocine_dataset)
+- **Paper:**
+- **Leaderboard:**
+- **Point of Contact:** [Théophile Blard](mailto:theophile.blard@gmail.com)
+
+### Dataset Summary
+
+The Allociné dataset is a French-language dataset for sentiment analysis. The texts are movie reviews written between 2006 and 2020 by members of the [Allociné.fr](https://www.allocine.fr/) community for various films. It contains 100k positive and 100k negative reviews divied into train (160k), validation (20k), and test (20k).
```
Comment (lhoestq) [inline datasets/allocine/README.md:64]:
```suggestion
The Allociné dataset is a French-language dataset for sentiment analysis. The texts are movie reviews written between 2006 and 2020 by members of the [Allociné.fr](https://www.allocine.fr/) community for various films. It contains 100k positive and 100k negative reviews divided into train (160k), validation (20k), and test (20k). 
```
small typo

**Review by sgugger — APPROVED** [review]
Looks great to me, thanks a lot!

**Pull request merged**